kornia/utils/grid.py

```diff
@@ -1,17 +1,15 @@
-from typing import Optional
-
 import torch
 
-from kornia.core import Tensor, stack
+from kornia.core import Device, Dtype, Tensor, stack
 from kornia.utils._compat import torch_meshgrid
 
 
 def create_meshgrid(
     height: int,
     width: int,
     normalized_coordinates: bool = True,
-    device: Optional[torch.device] = torch.device('cpu'),
-    dtype: torch.dtype = torch.float32,
+    device: Device = torch.device('cpu'),
+    dtype: Dtype = torch.float32,
```

Comment:
this isn't supported by JIT, and makes some tests crash, I suggest keeping it untouched, and dropping the JIT support separately (#2200)

Reply:
ok, I will revert the change. I will also need to change the typing for signature of `generate_anchors()` (in rtdetr_head.py) and `build_2d_sincos_pos_emb()` (in hybrid_encoder.py) since mypy will complain.